### PR TITLE
Hotels promotions

### LINKED ISSUE #80: Negative number
In the inputs, search hotel, add the validation in `minimum price`,` maximum price` and `number of beds` so that it does not accept negative numbers.

### PR DESCRIPTION
**Pull request by paulotijero**
Resolve #80 #5

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -8,11 +8,15 @@ def index
...
   
   def search
-    # it is necessary to validate the rented roms and not show them in the list
+    # TODO: it is necessary to validate the rented roms and not show them in the list
     query = {}
     query[:price] = params[:price_min]..params[:price_max] if params[:price_max].present? && params[:price_min].present? 
     query[:amount_of_beds] = params[:num_beds] if params[:num_beds].present?
     @rooms = Room.joins(:hotel).where(query)
     @rooms = @rooms.where("hotels.city ILIKE ?", "%#{params[:destiny]}%")  if params[:destiny].present?
   end
```
Comment (liannivin) [inline app/controllers/home_controller.rb]: Could you explain what this query do, please?

Comment (paulotijero) [reply app/controllers/home_controller.rb]: I think query resolve the search rooms for parameters enter, but, for more details, @condef5 is the person

File: app/views/home/promotions.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<div>
+  <hr>
+  <% @hotels.each do |hotel| %>
+    <h4>Hotel: <%= hotel.name %></h4>
+    <h4>Pais: <%= hotel.country %></h4>
+    <h4>Ciudad: <%= hotel.city %></h4>
+    <h2> 
+      Hasta <%= hotel.promotions.last.discount_amount %> 
+        <% if hotel.promotions.last.discount_type == "percentage"%>
+          %
```
Comment (liannivin) [inline app/views/home/promotions.html.erb]: Hey it seems as a little error.

File: app/views/partials/_search.html.erb [hunk 3]
```diff
@@ -2,8 +2,8 @@
...
   <%= date_field_tag :start_date, params[:start_date], placeholder: 'Start Date' %>
   <%= date_field_tag :end_date, params[:end_date], placeholder: 'Start Date' %>
-  <%= number_field_tag :price_min, params[:price_min], placeholder: 'Min price' %>
-  <%= number_field_tag :price_max, params[:price_max], placeholder: 'Max price' %>
-  <%= number_field_tag :num_beds, params[:num_beds], placeholder: 'Num of beds' %>
+  <%= number_field_tag :price_min, params[:price_min], placeholder: 'Min price', min:'0' %>
+  <%= number_field_tag :price_max, params[:price_max], placeholder: 'Max price', min:'0' %>
+  <%= number_field_tag :num_beds, params[:num_beds], placeholder: 'Num of beds', min:'0' %>
   <%= submit_tag "Search", :name => nil %>
 <% end %>
```
Comment (liannivin) [inline app/views/partials/_search.html.erb]: Nice!

File: app/views/home/promotions.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        <% else %>
+          Soles
+        <% end %>
+      de descuento 
+    </h2>
+    <p>Direccion: <%= hotel.address %></p>
+    <p>email: <%= hotel.email %></p>
+    <hr>
+  <% end %>  
+</div>
```
Comment (liannivin) [inline app/views/home/promotions.html.erb]: I like this!

**Pull request merged**